src/hotspot/share/runtime/globals.hpp

```diff
@@ -647,6 +647,9 @@ const int ObjectAlignmentInBytes = 8;
           "Show exception messages from RuntimeExceptions that contain "    \
           "snippets of the failing code. Disable this to improve privacy.") \
                                                                             \
+  product(bool, ForceLoadDisassembler, false, MANAGEABLE,                   \
+          "Always loading hotspot disassembler")                            \
```

Comment:
That doesn't quite describe what the flag does. I would suggest calling it `RetryLoadDisassembler` with a description:
```
Always try to reload the disassembler, even if a previous load failed"
```